examples/makefile

```diff
@@ -18,53 +18,98 @@ CASES			= ${TRAINING_CASES} ${TESTING_CASES}
 
 setup: dir mesh plot_config
 
+# Create the directory structure
+# ==============================
+#
+#   $(MODEL)
+#      ├── data
+#      ├── outputs
+#      │     └── $(TESTING_CASES)
+#      └── plots
 dir:
 	mkdir -p $(MODEL)/data
 	mkdir -p $(MODEL)/outputs
 	mkdir -p $(MODEL)/plots
-	touch $(MODEL)/__init__.py
-	for test_case in $(TESTING_CASES); do \
-		mkdir -p $(MODEL)/outputs/$$test_case; \
+	for case in $(TESTING_CASES); do \
+		mkdir -p $(MODEL)/outputs/$$case; \
 	done
 
+# Generate meshes
+# ===============
+#
+# Meshes are generated for all training and testing cases.
+#  * First, a gmsh geometry file is generated using the
+#    `meshgen.py` script.
```

Comment:
Mention here that the testing cases are randomly generated according to `$(MODEL)/config.py`.